Makefile

```diff
@@ -1,5 +1,7 @@
 GO ?= go
 
+export GOPROXY=https://proxy.golang.org
```

Comment:
Do we also want to set `GOSUMDB=sum.golang.org`?

Reply:
Sounds good to me. However, it'll only take effect on go >= 1.13.